src/examples/rest/.gitignore

```diff
@@ -1,3 +1,4 @@
 # Working build files
 .esbuild
 .graphweaver
+*.pem
```

Comment:
What is this?

Reply:
Authorisation has been added to the REST example. If you follow the steps in the README.txt, it will generate .pem files.